### PR TITLE
HBK.MicrophoneViewer version 7.0.0

### PR DESCRIPTION
**Pull request by lars-nvhgroup**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141207)

### CODE REVIEW

**Comment by lars-whta** [comment]
www.bksv.com is the proper and official domain for the package in question.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Edit: My changes became lost.

File: manifests/h/HBK/MicrophoneViewer/7.0.0/HBK.MicrophoneViewer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: HBK.MicrophoneViewer
+PackageVersion: 7.0.0
+PackageLocale: en-US
+Publisher: Brüel & Kjær
+PackageName: Brüel & Kjær Microphone Viewer
```
Comment (stephengillie) [inline manifests/h/HBK/MicrophoneViewer/7.0.0/HBK.MicrophoneViewer.locale.en-US.yaml:8]:
```suggestion
PackageName: Brüel & Kjær Microphone Viewer
PackageUrl: https://www.bksv.com/en/services/downloads/microphone-viewer/microphone-viewer
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This appears to be the correct PackageUrl for this package. Please update if there is a better PackageUrl.

**Comment by stephengillie** [comment]
Hi @lars-whta,

Generally, any user should be able to navigate from the PackageUrl to the InstallerUrl. From the PackageUrl I found and suggested above, I was blocked from downloading by a registration form:

image

Is there a page on this website that provides the package for download without signing up? Our package manager doesn't really have a mechanism to perform user signups and validate web forms like this.

**Pull request closed without merging**